### PR TITLE
Configurable Interval for Debouncing Keystore Reloading in Validator Client

### PR DESCRIPTION
**Pull request by rauljordan**
No tracking issue. This feature adds the ability to configure the debounce interval used in the validator client to detect changes to keystores in the wallet on disk. Making it easier to import large chunks of keystores into the validator client dynamically without it triggering a reload every second. The flag is:

```
--dynamic-key-reload-debounce-interval
```
Measured in seconds and with a default value of 1.

### CODE REVIEW

**Comment by rkapka** [comment]
My understanding, taken from a few Discord discussions, is that we don't want to flood users with flags. If this is right, is this flag worth adding? If I import a thousand keystores, does it have a negative effect on the validator other than outputting a bunch of logs? Also, this scenario is probably pretty unlikely.

File: shared/featureconfig/config.go [hunk 1]
```diff
@@ -66,6 +66,10 @@ type Flags struct {
 
 	KafkaBootstrapServers          string // KafkaBootstrapServers to find kafka servers to stream blocks, attestations, etc.
 	AttestationAggregationStrategy string // AttestationAggregationStrategy defines aggregation strategy to be used when aggregating.
+
+	// KeystoreImportDebounceInterval specifies the time duration the validator waits to reload new keys if they have
+	// changed on disk. This feature is for advanced use cases only.
+	KeystoreImportDebounceInterval int
```
Comment (prestonvanloon) [inline shared/featureconfig/config.go:72]: What about making this a time.Duration?

File: shared/featureconfig/config.go [hunk 2]
```diff
@@ -223,6 +227,7 @@ func ConfigureValidator(ctx *cli.Context) {
 		log.Warn("Disabling new BLS library blst")
 		cfg.EnableBlst = false
 	}
+	cfg.KeystoreImportDebounceInterval = ctx.Int(dynamicKeyReloadDebounceInterval.Name)
```
Comment (prestonvanloon) [inline shared/featureconfig/config.go:230]: We can use ctx.Duration and have dynamicKeyReloadDebounceInterval to be a DurationFlag. 

https://pkg.go.dev/github.com/urfave/cli/v2#Context.Duration

File: shared/featureconfig/flags.go [hunk 3]
```diff
@@ -94,6 +94,11 @@ var (
 		Usage: "(Danger): Disables the cache for attesting history in the validator DB, greatly increasing " +
 			"disk reads and writes as well as increasing time required for attestations to be produced",
 	}
+	dynamicKeyReloadDebounceInterval = &cli.IntFlag{
```
Comment (prestonvanloon) [inline shared/featureconfig/flags.go:97]: DurationFlag

**Review by prestonvanloon — CHANGES_REQUESTED** [review]
Using a DurationFlag removes the ambiguity of time units and allows for more flexible configuration.

Users could do `--dynamic-key-reload-debounce-interval=1s` or `--dynamic-key-reload-debounce-interval=1500ms` which is more clear than `--dynamic-key-reload-debounce-interval=1.5`.

**Comment by rauljordan** [comment]
Excellent suggestion, thank you

**Review by prestonvanloon — APPROVED** [review]

**Pull request merged**